source/content/style-guide.md

```diff
@@ -1078,6 +1078,42 @@ export env=dev
 
 </Example>
 
+## Videos
+Use the Wistia and YouTube components to embed videos across the docs site.
+
+With the exception of landing pages (e.g., [Go Live](/go-live), [Optimize Performance](/performance), etc.) which are sourced from [`source/data/landings.yaml`](https://github.com/pantheon-systems/documentation/blob/main/source/data/landings.yaml) and only support YouTube video embedding.
```

Comment:
@rachelwhitton this sentence seems to be missing a word or two. I'm not sure what it is saying.